common/djangoapps/util/views.py

```diff
@@ -75,8 +75,8 @@ def require_global_staff(func):
     @wraps(func)
     def wrapped(request, *args, **kwargs):
         edly_user_info_cookie = request.COOKIES.get(settings.EDLY_USER_INFO_COOKIE_NAME, None) if request else None
-        edly_sub_org = get_edly_sub_org_from_cookie(edly_user_info_cookie)
-        if GlobalStaff().has_user(request.user) or GlobalCourseCreatorRole(edly_sub_org).has_user(request.user):
+        edx_orgs = get_edx_org_from_cookie(edly_user_info_cookie)
+        if GlobalStaff().has_user(request.user) or GlobalCourseCreatorRole(edx_orgs[0]).has_user(request.user):
```

Comment:
`edx_orgs[0]` might break if list is empty

Reply:
Check added